### ISSUE TITLE
Bootstrap 4 support?

### ISSUE THREAD

**Issue by derekphilipau** [ARCANEDEV/LogViewer#169]
- LogViewer Version: 4.4.0
- Laravel Version: 5.5
- PHP Version: 7.1

### Description:

I know it's early and still beta, but are there plans to update templates to Bootstrap 4 in the near future?

**Comment by arcanedev-maroc** [comment]
You can submit a PR for this feature 👍 

> Try to implement multiple templates (config/views) for backward compatibility: `bootstrap-4, bootstrap-3 ...`

**Comment by derekphilipau** [comment]
It's a great idea to add "themes".  Today I made a working branch:  https://github.com/derekphilipau/LogViewer  I'm not sure if you will agree with the way I have done things.  The resources/views now contains two directories, "bootstrap-3" (original) and "bootstrap-4". 
 The "theme" is set in config/log-viewer.php, currently either "bootstrap-3" (original) or "bootstrap-4".  In LogViewerServiceProvider's boot() I added code to get the config theme and then call $this->loadViewsFrom(...);  While I tried to stay true to your design, the bootstrap-4 version is very vanilla and uses minimal styling.  I will ask a friend to double-check my code and then try a PR.  Thanks.
